downstream/modules/platform/con-aap-migration-prepare.adoc

```diff
@@ -0,0 +1,7 @@
+[id="aap-migration-prepare"]
+
+= Preparing for migration
+
+
+[role="_abstract"]
+Before migrating your current {PlatformNameShort} deployment to {OperatorPlatform}, you need to backup your existing data, create secret keys, and create a postresql configuration secret for migration.
```

Comment:
Instead of:
```
create secret keys, and create a postresql configuration secret
```

This is a bit more accurate and explicit:

```
create k8s secrets for your secret key and postgresql configuration
```

It gets confusing because the kubernetes construct for storing sensitive data called a "Secret".  So the user has to create a k8s "Secret" containing the secret key value (also known as the symmetric key) for the old T

Reply:
I second this, the second version is much more descriptive.

Reply:
Suggested changes complete.